apis/swagger.yml

```diff
@@ -811,6 +811,22 @@ paths:
         - $ref: "#/parameters/id"
       tags: ["Network"]
 
+  /networks:
+      get:
+        summary: "List networks"
+        operationId: "NetworkList"
+        produces: ["application/json"]
+        responses:
+          200:
+            description: "Summary networks that matches the query"
+            schema:
+              type: "array"
+              items:
+                $ref: "#/definitions/NetworkListResp"
```

Comment:
I do not think this correct. Since in code you returned `respNetworks := types.NetworkListResp{Networks: []*types.NetworkInfo{}, Warnings: nil}`. But here you returned `[]types.NetworkListResp`. They are not the same.

You have already defined  `Networks` in `NetworkListResp` as an array. So you do not need to make this api to return an array of NetworkListResp. This is not correct, to be honest. @ZouRui89